doc/API-mapping.md

```diff
@@ -171,6 +171,33 @@ Now, the API call URLs will look like:
 
 http://your.api.domain/entityName?q=foo&tag=bar
 
+
+## Nested request and response
+
+If your API returns the results in a field of an object instead directly, you can use Restangular response interceptor. For instance if you request http://your.api.domain/entityName and the API returns the result array in an `items` field instead directly:
+
+```js
+app.config(function(RestangularProvider) {
+    RestangularProvider.addResponseInterceptor(function(data, operation, what, url, response) {
+        if (operation == "getList" && what == 'entityName') {
+            data = data.items;
+        }
+        return data;
+});
+```
+
+If your API requires that you post and put data for instance inside of an `item` field instead directly, use Restangular request interceptor:
```

Comment:
Good, but you shouldn't add a new section about it, "Entry Format" is there for it.